### PR TITLE
Domain Ownership Cleanup

### PR DESCRIPTION
**Pull request by camhashemi**
@leogrim @ryanpbrewster 

also releases the RHR UI from the experiment and adds a test for verify-to-join

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/profile/profileWidget.tpl.html [hunk 1]
```diff
@@ -99,7 +99,7 @@
       </div>
       <button class="kf-pw-org-btn kf-pw-org-btn-accept" ng-if="org.pending && !org.declined" ng-click="acceptInvite(org)">Accept</button>
       <button class="kf-pw-org-btn kf-pw-org-btn-decline" ng-if="org.pending && !org.declined" ng-click="declineInvite(org)">Decline</button>
-      <div class="kf-pw-org-email" ng-if="org.potential && org.viewer.emails && me.experiments.indexOf('admin') !== -1">
+      <div class="kf-pw-org-email" ng-if="org.potential && org.viewer.emails">
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/profile/profileWidget.tpl.html:102]: @cvializ releasing the RHR

File: kifi-backend/modules/shoebox/test/com/keepit/controllers/core/AuthControllerTest.scala [hunk 2]
```diff
@@ -89,5 +89,39 @@ class AuthControllerTest extends Specification with ShoeboxTestInjector {
...
+        val (newUserEmail, membershipOpt, membershipOpt2) = db.readOnlyMaster { implicit s =>
+          val userEmail = userEmailAddressRepo.getByAddress(EmailAddress("orangutan@primate.org")).get
+          val orgMembership = membershipRepo.getByOrgIdAndUserId(org.id.get, user.id.get)
+          val orgMembership2 = membershipRepo.getByOrgIdAndUserId(org2.id.get, user.id.get)
+          (userEmail, orgMembership, orgMembership2)
+        }
+
+        newUserEmail.verified === true
+        membershipOpt.isDefined === true
+        membershipOpt2.isDefined === true
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/core/AuthControllerTest.scala]: `must beSome` will give you nicer error messages

File: kifi-backend/modules/shoebox/angular/src/profile/profileWidget.tpl.html (hunk 1 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/profile/profileWidget.tpl.html:102]: :raised_hands: hooray!

File: kifi-backend/modules/shoebox/marketing [hunk 3]
```diff
@@ -1 +1 @@
-Subproject commit cc275c36e666674d995582050c0d36a050a268f2
+Subproject commit 16788b5d4cb97c8be710027d3bb31fa28cb5e4c7
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/marketing]: frustration

File: kifi-backend/modules/shoebox/test/com/keepit/controllers/core/AuthControllerTest.scala (hunk 2 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/core/AuthControllerTest.scala]: for some reason I couldn't get it to compile that way.

**Pull request merged**